asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -333,6 +357,24 @@ def set_expansion(
             parent=parent, provenance=provenance
         )
 
+    def has_stereo(self) -> bool:
+        """
+        Check if the ligand has any stereo bonds or chiral centers excluding nitrogen centers.
```

Comment:
Make the nitrogen bit an option as is a bit misleading.

Reply:
Good point I think we can drop the nitrogen exclusion for now and come back to it if its still an issue. I think this was another attempt at removing inconsistent stereo which should be fixed now.